packages/realm/src/tests/schema-normalization.test.ts

```diff
@@ -751,6 +751,21 @@ describe("normalizePropertySchema", () => {
       },
       { isPrimaryKey: true },
     );
+
+    itNormalizes(
+      {
+        type: "list",
+        objectType: "string",
+        indexed: true,
+      },
+      {
+        type: "list",
+        objectType: "string",
+        indexed: true,
+        optional: false,
+      },
+      { isPrimaryKey: false },
+    );
```

Comment:
Seems fine, but it also seems unrelated to the upgrade.

Reply:
I added this because the upgrade also included being able to index list of strings. So this tests that the parser also allows this.

Reply:
It is testing an enhancement mentioned in the changelog.